helm/charts/text-generation-inference/templates/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "text-generation-inference.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Haven't you specified in the Chart.yaml that it is compatible for k8s ^1.27 ?
(not saying we should change the code here, rather update the kubeversion in the Chart.yaml)

Reply:
the ingress definition is straight from `helm create`. the default definition is enough for the current state of the chart.
I've define k8s ^1.27 in Chart.yaml as this is the version I'm currently using for testing.
I can align everything to 1.27 if that makes more sense